tools-image/build-arm-image.sh

```diff
@@ -417,6 +424,9 @@ cp -rfv /defaults.yaml $WORKDIR/persistent/cloud-config/01_defaults.yaml
 
 grub2-editenv $WORKDIR/state/grub_oem_env set "default_menu_entry=Kairos"
 
+# We copy the file we saved earier to the STATE partition
+cp -rfv "${tmpgrubconfig}" $WORKDIR/state/grubmenu
```

Comment:
same is needed for the `prepare_arm_image.sh` script

Reply:
https://github.com/kairos-io/osbuilder/blob/a868346eacfd356f351da6248f3b4ab1dda4be2e/tools-image/prepare_arm_images.sh#L109

Reply:
maybe we can do a common script to prepare the state directory? at least its consistent between all the scripts here